superbench/benchmarks/model_benchmarks/pytorch_gpt2.py

```diff
@@ -99,9 +110,31 @@ def _create_model(self, precision):
             n_embd=self._args.hidden_size, n_layer=self._args.num_hidden_layers, n_head=self._args.num_attention_heads
         )
 
+        enable_fp8 = precision.name.startswith('FP8_')
+        if enable_fp8 and te is None:
+            logger.error(
+                f'Create model with fp8 failed - model: {self._name}, precision: {precision},'
+                ' message: Cannot find transformer_engine.'
+            )
+            return False
+        if enable_fp8 and not self._gpu_available:
+            logger.error(
+                f'Create model with fp8 failed - model: {self._name}, precision: {precision},'
+                ' message: FP8 is only supported on GPU.'
+            )
```

Comment:
Why _gpu_available is only for FP8?

Reply:
other precisions (fp16/fp32) can run on cpu but current te fp8 can only run on gpu, there's no cpu implementation